app/gateway/changelog.md

```diff
@@ -402,6 +402,11 @@ It has now been reverted back to `kong-enterprise-edition.service` to keep consi
   [#10766](https://github.com/Kong/kong/pull/10766)
 * Bumped `kong-openid-connect` from 2.5.4 to 2.5.5
 
+### Known Issues
+
+* Due to known issues, Kong recommends not enabling page-level lmbd encryption in versions 3.0.x-3.3.x. 
+  
+  Don't set `declarative_config_encryption_mode`; leave it at its default value of `off`. Continue relying on disk-level encryption to encrypt the configuration on disk.
```

Comment:
Hi @lena-larionova ,

Could you also add the following items to the known issues list?

```
# Known Issues
* Sending an invalid configuration to the `/config` endpoint while running in dbless mode and with `flatten_errors=1` set, Kong incorrectly returns a 500. This should be a 400 because the configuration is invalid.
* An OIDC plugin configured to reference HashiCorp Vault in the `config.client_secret` field (EG, `{vault://hcv/clientSecret}`) does not look up the secret correctly.
```